dev-server/mock-routes.js

```diff
@@ -33,7 +33,12 @@ app.get('/familie-ef-sak/api/personinfo', (req, res) => {
     setTimeout(() => res.send(lesMockFil(filnavn)), delayMs);
 });
 
-app.post('/familie-ef-sak/api/saksoek/ident', (req, res) => {
+app.get('/familie-ef-sak/api/sak/:id', (req, res) => {
```

Comment:
Er dette et tenkt endepunkt for å hente ut sak gitt id på sak?

Reply:
Bør være /fagsaker/:fagsakId

Reply:
Dette endepunktet finnes og er det tynneste halmstrået vi har til en sak enn så lenge. Vi vil jo ha en tilstand basert på en sak, ikke personident - følgelig bruker jeg dette inntil videre. Men vi må nok gå opp løypa for hvordan dette skal være i praksis. Hvis jeg forstod det riktig kan vi ha 3 saker per person - isåfall blir nok utgangspunktet vårt noe annerledes enn f.eks. BA.

Reply:
:O :O Hvorfor?

Reply:
> På generell basis er innholdet her veldig bra. Jeg tenker kanskje at vi bør ta noen skritt tilbake og trekke ut noen av komponentene her til felles. Er veldig mye likt og dumt om vi sklir ut allerede her. Ser at det er noen forskjeller allerede som kanskje kan være nyttig for oss.

Jeg er enig i at noe av dette nok faller inn under felles-begrepet, men samtidig ønsker jeg å få avklart litt mer hvordan det skal være før jeg dytter det inn i felleskomponentene for å unngå endringer frem-og-tilbake. 

På samme måte kan det være dere har laget noen gull-komponenter i BA som egentlig passer godt som en standalone-komponent på tvers for begge parter.

Reply:
> :O :O Hvorfor?

Det ble nevnt i en prat i EF-sak-gjengen tidligere at hver stønad skulle behandles hver for seg og være "uavhengig" av hverandre, men kan nok godt hende vi teknisk kan samle det som én sak. 

Det hadde nok vært på sin plass å få tatt en prat med deg og noen andre fra BAKS for å gå igjennom hva som kan funke/ikke funke i EF-øyemed. Ikke sikkert det er lurt å sitte her og finne på kruttet på nytt :)

Reply:
https://www.nav.no/soknader/nb/person/familie/enslig-mor-eller-far
Overgangsstønad, skolepenger og barnetilsyn er egne "fagsaker" og kan søkes om og behandles uavhengig av hverandre. Mye overlapp - og avhengigheter mellom dem er det riktignok.